nncf/api/composite_compression.py

```diff
@@ -23,6 +24,38 @@
 ModelType = TypeVar('ModelType')
 
 
+class CompositeStatistics(Statistics):
```

Comment:
If we look in terms of the main usage scenarios.  I guess, It is access to statistics values. We will see that the `CompressionAlgorithmController` returns `Statistics` object, but the `CompositeCompressionAlgorithmController` returns `CompositeStatistics` object. Currently, If the developer want to get sparsity_level, for example, then he should write the following code:

```
statistics = compression_ctrl.statistics()
if isinstance(statistics, CompositeStatistics):
    items = statistics.child_statistics
else:
    items = [statistics]

for item in items:
    if isinstance(item, (MagnitudeSparsityStatistics, RBSparsityStatistics, ConstSparsityStatistics)):
          sparsity_level = item.model_statistics.sparsity_level
    
```
before these changes it was like this: 

```
statistics = compression_ctrl.statistics()
sparsity_level = statistics['sparsity_level']
```

I was expecting the following:

```
statistics = compression_ctrl.statistics()
sparsity_level = statistics.magnitude_sparsity.sparsity_level
#or
magnitude_sparsity_statistics = statistics.get_statistics('magnitude_sparsity')
sparsity_level = magnitude_sparsity_statistics.sparsity_level
```
Can we make the statistics method return one type of object? Statistics container? @andrey-churkin, @vshampor Could you look once more at the proposal in this PR in terms of UX/DX?

Reply:
Thank you very much for your suggestion. I fixed it in b5398b0.